frontend/src/pages/eventOverview.js

```diff
@@ -74,105 +48,27 @@ class EventOverview extends Component {
       description: this.state.description,
       fbLink: this.state.fbLink
     }
-    const { code } = await createEvent(event)
-    if (code === 500) {
-      this.setState({
-        alert: 'All fields are required.'
-      })
-    } else {
-      Router.push({ pathname: '/eventOverview' })
-    }
+    const { success } = await createEvent(event)
```

Comment:
nice use of passing functions as props, this makes the modal even more extendable in the future if we want to use it for other things.

Reply:
thanks!